core/src/main/kotlin/com/exactpro/th2/read/db/app/DataBaseReaderConfiguration.kt

```diff
@@ -56,7 +56,7 @@ data class ReadTaskConfiguration(
 @JsonTypeName("pull")
 data class PullTaskConfiguration(
     val dataSource: DataSourceId,
-    val initQueryId: QueryId,
+    val initQueryId: QueryId?,
```

Comment:
Please, modify the gRPC implementation as well. It should correctly process the pulling task request without init query set